dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task201102UpdateColumnSitelicTable.java

```diff
@@ -45,9 +45,8 @@ public void executeUpgrade() throws DotDataException, DotRuntimeException {
             throw new DotDataException(e.getMessage(), e);
         }
 
-        final String columnLiteral = DbConnectionFactory.isMsSql() ? "" : "COLUMN";
         new DotConnect()
-                .setSQL(String.format("ALTER TABLE sitelic ADD %s startup_time %s", columnLiteral, type.get()))
+                .setSQL(String.format("ALTER TABLE sitelic ADD startup_time %s", type.get()))
```

Comment:
Postgres and Mysql still need the 'COLUMN' keyword.

Reply:
No, those neither need them, look old UT https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05305AddPushPublishFilterColumn.java

Reply:
Nevermind it seems that we never specify that keyword for any of the databases when it comes to adding a column.